packages/vite/src/node/build.ts

```diff
@@ -702,10 +706,22 @@ export async function build(
     for (const output of normalizedOutputs) {
       res.push(await bundle[options.write ? 'write' : 'generate'](output))
     }
+    const shouldLogInfo = LogLevels[config.logLevel || 'info'] >= LogLevels.info
+    if (shouldLogInfo) {
```

Comment:
I don't think we need this check here. `config.logger.info` already checks this internally.

Reply:
Yes, you are right, I ported directly from `reporter.ts` and didn't notice this, thanks.